CPRynner/CPRynner.py

```diff
@@ -242,17 +220,22 @@ def cluster_tasks_per_node():
         tasks_per_node = ''
     return tasks_per_node
 
-
```

Comment:
Two blank lines between function definitions (not methods, however) is a common stylistic convention in Python.